content/installation/netcore/README.md

```diff
@@ -0,0 +1,20 @@
+The .NET Core doc structure is based on our existing .NET docs.  There would be a new section for .NET Core, setup as below:
+
+.NET Core
+
+  * Overview
+    - /netcore/install/install-overview-netcore.md
+  * System Requirements 
+    - /netcore/install/System-requirements-netcore.md
+  * Supported Technologies
+    - /netcore/install/Supported-technology-netcore.md
+  * Installation 
+    - /netcore/install/NetCoreInstallation.md
+  * Usage:         
+    - /netcore/usage/GeneralUsage.md
+    - /netcore/usage/UsingDotNetAzure.md
+    - /netcore/usage/UsingEithIISExpress.md
+  * Configuration:
+    - Same as .NET: link to the same page
```

Comment:
Does this include the General Settings, Application-Specific Settings and Settings for Azure App Service articles for the .NET agent?

Reply:
No - just the YAML Properties and YAML Template section.  I guess this should be a separate doc then?